### ISSUE TITLE
Validation set score visualization

### ISSUE THREAD

**Issue by ZhongBaby** [deeplearning4j/deeplearning4j#1058]
I want to use this in order to set some anti-overfitting hyperparameters.
For now I can see score during model training according to the listener frequency.
So it looks like the image below.(courtesy of http://neuralnetworksanddeeplearning.com/chap3.html)
![train](https://cloud.githubusercontent.com/assets/8222739/12448577/e4892d02-bfb1-11e5-81c4-1c4eb976ed4a.png)

And the validation set score maybe like the image below.(courtesy of http://neuralnetworksanddeeplearning.com/chap3.html)
![test](https://cloud.githubusercontent.com/assets/8222739/12448620/3cb1ebe0-bfb2-11e5-8b5e-407cf0d8d9a9.png)

I want a blue line for training listener, and simultaneously plot a orange line for validation set, note validation set is not used to train the model, so there needs some change not only by the UI.
As the image below(courtesy of me :smile: )
![123](https://cloud.githubusercontent.com/assets/8222739/12448765/410dc406-bfb3-11e5-962c-aa84b9d5583a.png)

When I see blue line decrease and orange line increase I know there is overfitting, and I should change my anti-overfitting hyperparameter such as L2 regularization bigger.

**Comment by agibsonccc** [comment]
New UI. Not relevant.
